spec-conventions.md

```diff
@@ -0,0 +1,46 @@
+# Conventions when writing specification text
+
+> **NOTE:** This document is currently a skeleton, and should not be considered a
+> complete list.
+> The issue [tc39/ecma262#2236](https://github.com/tc39/ecma262/issues/2236) is
+> a good place to discuss conventions.
+
+## Spelling, grammar, and typography
+
```

Comment:
```suggestion
## Spelling, grammar, and typography

Specification text should use Oxford spelling (e.g., "behaviour" rather than "behavior").
```

Reply:
No, this is an editorial convention, which we will document on the ecma262 wiki soon™️. See https://github.com/tc39/how-we-work/pull/119#issuecomment-1326796200